packages/kbn-unified-data-table/src/components/data_table.tsx

```diff
@@ -1037,6 +1037,7 @@ export const UnifiedDataTable = ({
           data-test-subj="discoverDocTable"
           data-render-complete={isRenderComplete}
           data-shared-item=""
+          data-rendering-count={1} // TODO: Fix this as part of https://github.com/elastic/kibana/issues/179376
```

Comment:
This used to be handled by the `RenderCompleteDispatcher`, but new React embeddables no longer use this - however, since legacy embeddables are still using the `RenderCompleteDispatcher`, we cannot remove it until **all** embeddables are converted to the new React embeddable system. Therefore, this will be updated to be the **actual** render count in https://github.com/elastic/kibana/issues/179376.